Oqtane.Server/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": ""
+    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane.mdf;Initial Catalog=Oqtane;Integrated Security=SSPI;"
```

Comment:
This should  be empty. You may have forgot to delete your connection string.

Reply:
I know, I figured I would remove it after the code was reviewed, because I was pretty sure it wouldn't be accepted right off the bat.